torch/_inductor/decomposition.py

```diff
@@ -385,6 +386,41 @@ def bernoulli(self, *, generator=None):
     return torch.rand_like(self, dtype=torch.float32) < self
 
 
+@register_decomposition([aten.fmin, prims.fmin])
+def fmin(self, other):
+    return torch.where(torch.isnan(other) | (other > self), self, other)
+
+
+@register_decomposition([aten.fmax, prims.fmax])
+def fmax(self, other):
+    return torch.where(torch.isnan(other) | (other < self), self, other)
+
+
+@register_decomposition([aten.narrow_copy])
+def narrow_copy(self, dim, start, length):
+    return torch.narrow(self, dim, start, length).clone()
+
+
+@register_decomposition([aten.expand_copy])
+def expand_copy(self, size, *, implicit=False):
+    return aten.expand(self, size, implicit=implicit).clone()
+
+
+@register_decomposition([aten.view_copy.default])
+def view_copy_default(self, size):
+    return aten.view(self, size).clone()
+
+
+@register_decomposition([aten.view_copy.dtype])
+def view_copy_dtype(self, dtype):
+    return self.to(dtype).clone()
+
+
+@register_decomposition([aten.nansum])
+def nansum(self, dim=None, keepdim=False, *, dtype=None):
+    return aten.sum(self.nan_to_num(), dim, keepdim, dtype=dtype)
```

Comment:
nan_to_num (somewhat unexpectedly) by default replaces infs with maximum/minimum representable values for dtype, that we don't want. Otherwise, this decomp also looks not inductor-specific, so can go to `_decomps`

Reply:
fixed